test/reference.html

```diff
@@ -907,14 +990,28 @@
                 }
                 j += 2;
             }
-        } else {
+        } else if (sections[i + 2] === 'wide') {
+            while (j < entries.length) {
+                if (entries[j] === sections[i]) {
+                    // entry matches the current section
+                    t += '<div class="wide-entry">';
+                    t += '<div><code>' + entries[j + 1] + '</code></div>';
+                    t += '<div class="mathquill-static-math">' + entries[j + 1] + '</div>';
+                    t += '<div class="codecogs"><img src="' + 'http://latex.codecogs.com/svg.latex?' + encodeURI(entries[j + 1]) + '"></div>';
+                    t += '<div class="mathjax">$$' + entries[j + 1] + '$$</div>';
+                    t += '</div>';
+                }
+                j += 2;
+            }
+       } else {
             t += '<table>';
             while (j < entries.length) {
                 if (entries[j] === sections[i]) {
                     // entry matches the current section
                     t += '<tr>';
                     t += '<td class="mathquill-static-math">' + entries[j + 1] + '</td>';
-                    t += '<td><img src="' + 'http://latex.codecogs.com/svg.latex?' + encodeURI(entries[j + 1]) + '"></td>';
+                    t += '<td class="codecogs"><img src="' + 'http://latex.codecogs.com/svg.latex?' + encodeURI(entries[j + 1]) + '"></td>';
+                    t += '<td class="mathjax">$$' + entries[j + 1] + '$$</td>';
```

Comment:
Damn, you're totally right

Han

On Mon, Apr 11, 2016 at 12:16 PM, Arno Gourdol <notifications@github.com>
wrote:

> In test/reference.html
> <https://github.com/mathquill/mathquill/pull/615#discussion_r59263817>:
>
> >              t += '<table>';
> >              while (j < entries.length) {
> >                  if (entries[j] === sections[i]) {
> >                      // entry matches the current section
> >                      t += '<tr>';
> >                      t += '<td class="mathquill-static-math">' + entries[j + 1] + '</td>';
> > -                    t += '<td><img src="' + 'http://latex.codecogs.com/svg.latex?' + encodeURI(entries[j + 1]) + '"></td>';
> > +                    t += '<td class="codecogs"><img src="' + 'http://latex.codecogs.com/svg.latex?' + encodeURI(entries[j + 1]) + '"></td>';
> > +                    t += '<td class="mathjax">$$' + entries[j + 1] + '$$</td>';
>
> I've switched to deferring the typesetting and making it explicit, but I
> don't see any difference in speed. Not sure why you think it would be
> faster. Scanning all the nodes to find the $$ is negligible compared to the
> cost of manipulating the DOM and redoing the layout.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/mathquill/mathquill/pull/615/files/10ce7424d9558525a74ad1ef00fca9c61248a8d2..6810806ca08bcb5f8cc1d3b3ce9e29465df38b93#r59263817>
>